modules/aws/vpc.go

```diff
@@ -39,30 +39,60 @@ func GetDefaultVpc(t *testing.T, region string) *Vpc {
 
 // GetDefaultVpcE fetches information about the default VPC in the given region.
 func GetDefaultVpcE(t *testing.T, region string) (*Vpc, error) {
-	client, err := NewEc2ClientE(t, region)
-	if err != nil {
-		return nil, err
+	defaultVpcFilter := ec2.Filter{Name: &isDefaultFilterName, Values: []*string{&isDefaultFilterValue}}
+	vpcs, err := GetVpcsE(t, []*ec2.Filter{&defaultVpcFilter}, region)
+
+	numVpcs := len(vpcs)
+	if numVpcs != 1 {
+		return nil, fmt.Errorf("Expected to find one default VPC in region %s but found %s", region, strconv.Itoa(numVpcs))
 	}
 
-	defaultVpcFilter := ec2.Filter{Name: &isDefaultFilterName, Values: []*string{&isDefaultFilterValue}}
-	vpcs, err := client.DescribeVpcs(&ec2.DescribeVpcsInput{Filters: []*ec2.Filter{&defaultVpcFilter}})
+	return vpcs[0], err
+}
+
+func GetVpcById(t *testing.T, vpcId string, region string) *Vpc {
+	vpc, err := GetVpcByIdE(t, vpcId, region)
 	if err != nil {
-		return nil, err
+		t.Fatal(err)
 	}
```

Comment:
We have a new pattern for this now, where we use `require.NoError(t, err)`. I know this file hasn't been updated to use the new pattern yet, but we like to update whenever we notice the old pattern. Can you go ahead to update this here, along with adding in the relevant import? Thanks!

Reply:
done